java/src/jmri/jmrit/display/palette/PaletteBundle.properties

```diff
@@ -59,7 +59,7 @@ ReadMemory          = Display Memory
 ReadWriteMemory     = Input Box Memory
 SpinnerMemory       = Spinner Memory
 ComboMemory         = ComboBox Memory
-ReadGlobalVariable          = Display Memory
+ReadGlobalVariable          = Display Global Variable
 ReadWriteGlobalVariable     = Input Box Memory
```

Comment:
I think you need to do the other 3 also.

Reply:
All four are fixed now.